packages/firestore/test/integration/api/query.test.ts

```diff
@@ -555,6 +557,58 @@ apiDescribe('Queries', persistence => {
     });
   });
 
+  it('can use IN filters', async () => {
+    const testDocs = {
+      a: { zip: 98101 },
+      b: { zip: 91102 },
+      c: { zip: 98103 },
+      d: { zip: [98101] },
+      e: { zip: ['98101', { zip: 98101 }] },
```

Comment:
nice negative cases! :)

Reply:
Inspired by your tests for array-contains!